duckduckgo/iOS
# Disable the safari content blocker

Reviewer: Chris
Asana: https://app.asana.com/0/361428290920652/394872926116311

**Description**:
Disable the safari content blocking extension.

**Steps to test this PR**:
_Ensure tutorial is removed:_
1. Go to the content blocker panel
2. Ensure the "Use Content Blocker in Safari" option is gone

_Ensure extension is removed from build:_
1. Open Safari settings
2. Go to Content Blockers
3. Ensure the DuckDuckGo content blocker is not in the list

###### Reviewer Checklist:


###### PR DRI Checklist:
